src/main/java/ooo/oshi/foreign/windows/Kernel32Library.java

```diff
@@ -60,5 +66,29 @@ public static int getCurrentProcessId() {
 
     private static final MethodHandle getCurrentProcessId = methodHandle("GetCurrentProcessId",
             FunctionDescriptor.of(JAVA_INT));
+    
+    /**
+     * Retrieves the NetBIOS name of the local computer. This name is established at system startup, when the system
+     * reads it from the registry.
+     *
+     * @return the NetBIOS name of the local computer.
+     */
+    public static String getComputerName() {
+    	try {
+        	SegmentAllocator allocator = SegmentAllocator.implicitAllocator();
+    		Addressable buffer = allocator.allocate(WinNT.MAX_COMPUTERNAME_LENGTH + 1);
```

Comment:
Looks good but we don't have a WInNT class here.   So you'll need to make one (or possibly a WinBase class that has all these headers).

Reply:
My bad :( I did implement it locally, but somehow missed it while pushing the changes.